lib/Elastica/Client.php

```diff
@@ -716,4 +716,27 @@ public function setLogger(LoggerInterface $logger)
 
         return $this;
     }
+
+    /**
+     * @param Script $script
+     * @param $lang
+     * @param $id
+     *
+     * @return Response
+     */
+    public function addIndexedScript(Script $script, $lang, $id)
+    {
+        return $this->request(sprintf('_scripts/%s/%s', $lang, $id), Request::POST, $script->toArray());
```

Comment:
Can you use string concatenation here? `'_script/' . $lang . '/' . $id)` ? The main reason is that it is used like this across the library.